docs/atomic-structure.md

```diff
@@ -192,24 +190,9 @@ cfgov-refresh/test/unit_tests/molecules/
 cfgov-refresh/test/unit_tests/organisms/
 ```
 
-#### Proposed Folder Structure
```

Comment:
This proposal should definitely get captured in a Platform issue or other appropriate location.

Reply:
Done - added to Platform GHE